docs/blog/posts/this-month-in-hax/2025-02.md

```diff
@@ -0,0 +1,62 @@
+---
+authors:
+  - lucas
+title: "This Month in Hax: February 2025"
+date: 2025-02-10
```

Comment:
```suggestion
date: 2025-03-05
```

Reply:
That's terrible how I always forget to update the date... I'm looking at it and think "oh yeah, that looks good, februrary" >.<

Reply:
Thanks!